tests/helpers.py

```diff
@@ -115,7 +117,7 @@ def reference_id_from_url(url):
     {
         **TEST_POLYGON_ANNOTATIONS[i],
         "confidence": 0.10 * i,
-        "class_pdf": TEST_POLYGON_MODEL_PDF,
```

Comment:
seems to me like we should keep this test pdf to ensure test coverage on the pdf upload feature.

Reply:
There was a bug I was testing here (when no class pdf, it would error out. BUt I think Jihan separately added a test case for that)